### PR TITLE
PXB-2960 8.0.31-24 Release notes and release note tasks (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
Changes to be committed:
	modified:   docs/copyright-and-licensing-information.md
	modified:   docs/release-notes.md
	new file:   docs/release-notes/8.0/8.0.31-24.0.md
	modified:   mkdocs-base.yml

### CODE REVIEW

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Review by rahulmalik87 — CHANGES_REQUESTED** [review]
Can you also mention about  https://jira.percona.com/browse/PXB-2925

**Comment by patrickbirch** [comment]
PXB-2925 is added.

File: docs/release-notes/8.0/8.0.31-24.0.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+This release adds the ability to use an IAM instance profile when running xbcloud from an EC2 instance. An IAM instance profile does not need the `--s3-secret-key` nor the `--s3-access-key` but the profile has a six-hour lifespan.
+ 
+## New features
+
+[PXB-2856](https://jira.percona.com/browse/PXB-2856): Use [IAM instance profile when running xbcloud from an EC2 instance](docs/xbcloud/xbcloud-iam-profile.md).
+
+## Bug fixes
+
+[PXB-2890](https://jira.percona.com/browse/PXB-2890): XtraBackup 8.0 inefficiently compressed already compressed tables.
```
Comment (altmannmarcelo) [inline docs/release-notes/8.0/8.0.31-24.0.md]: Its the other way around. It was not compressing which is inefficiently. It now compress compressed tables.

File: docs/release-notes/8.0/8.0.31-24.0.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+[PXB-2928](https://jira.percona.com/browse/PXB-2828): XtraBackup exits with signal 11 when taking a backup.
```
Comment (altmannmarcelo) [inline docs/release-notes/8.0/8.0.31-24.0.md]: Link is pointing to a different issue.

Comment (altmannmarcelo) [inline docs/release-notes/8.0/8.0.31-24.0.md]:
```suggestion
[PXB-2928](https://jira.percona.com/browse/PXB-2928): XtraBackup exits with signal 11 when taking a backup using page tracking.
```

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]
Please add:

[PXB-2964](https://jira.percona.com/browse/PXB-2964)
https://jira.percona.com/browse/PXB-2959 - PXB now compiles on platforms with openssl 3.0
https://jira.percona.com/browse/PXB-2971

File: docs/release-notes/8.0/8.0.31-24.0.md (hunk 1 above)
Comment (patrickbirch) [reply docs/release-notes/8.0/8.0.31-24.0.md]: Rewritten.

File: docs/release-notes/8.0/8.0.31-24.0.md (hunk 2 above)
Comment (patrickbirch) [reply docs/release-notes/8.0/8.0.31-24.0.md]: Fixed.

Comment (patrickbirch) [reply docs/release-notes/8.0/8.0.31-24.0.md]: Fixed.

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Review by rahulmalik87 — APPROVED** [review]

File: docs/release-notes/8.0/8.0.31-24.0.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        font-family: Chivo, Colfax, Franziska, Helvetica, Arial, sans-serif;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+    }
```
Comment (patrickbirch) [inline docs/release-notes/8.0/8.0.31-24.0.md:15]: Extra ( } ) before </style>

**Pull request merged**